packages/editor-ui/src/components/CodeNodeEditor/CodeNodeEditor.vue

```diff
@@ -322,7 +332,12 @@ export default defineComponent({
 			...readOnlyEditorExtensions,
 			EditorState.readOnly.of(isReadOnly),
 			EditorView.editable.of(!isReadOnly),
-			codeNodeEditorTheme({ isReadOnly, customMinHeight: this.rows }),
+			codeNodeEditorTheme({
+				isReadOnly,
+				maxHeight: this.fillParent ? '100%' : 'max(calc(100vh - 435px), 80px)', // 435px represents the height of all the various elements and padding in that page
```

Comment:
Can we get away without using magic numbers (`435px` or `515px`) here?

Reply:
Yeah changed it, this value works perfectly in the code node, but it's not a generic solution because it depends on the other stuff in the NDV. Changed to a simpler value.